metrics/processors/node_autoscaling_enricher.go

```diff
@@ -49,6 +49,7 @@ func (this *NodeAutoscalingEnricher) Process(batch *core.DataBatch) (*core.DataB
 			capacityMem, _ := node.Status.Capacity[kube_api.ResourceMemory]
 			allocatableCpu, _ := node.Status.Allocatable[kube_api.ResourceCPU]
 			allocatableMem, _ := node.Status.Allocatable[kube_api.ResourceMemory]
+			capacityNvidia, _ := node.Status.Capacity[kube_api.ResourceNvidiaGPU]
```

Comment:
ResourceNvidiaGPU is a deprecated resource and is going away. The new resources are exposed through device plugins. I would not add this metric at all for now.

Reply:
> ResourceNvidiaGPU is a deprecated resource and is going away. The new resources are exposed through device plugins.

Does the resources has a predefined or patterned names? @mindprince

Reply:
As for now, It seems that [`alpha.kubernetes.io/nvidia-gpu` is still available](https://github.com/kubernetes/api/blob/master/core/v1/types.go#L3840). So, i think it is fine to add this for now. 

WDYT @piosz @mindprince

Reply:
I have add a test for `ResourceNvidiaGPU` resource name, so even it is not supported and removed in the future, everything should work fine.